imgui/core.pyx

```diff
@@ -1935,6 +1935,220 @@ def menu_item(
     return clicked, inout_selected
 
 
+def open_popup(str name):
+    """Open a popup window.
+
+    Marks a popup window as open. Popups are closed when user click outside,
+    or activate a pressable item, or :func:`close_current_popup()` is
+    called within a :func:`begin_popup()`/:func:`end_popup()` block.
+    Popup identifiers are relative to the current ID-stack
+    (so :func:`open_popup` and :func:`begin_popup` needs to be at
+    the same level).
+
+    .. visual-example::
```

Comment:
Note that **popups** (defined with `BeginPopup`) are different from **popup modals** (defined with `BeginPopupModal`). We should have two examples:
* one showing simple popups in `begin_popup`
* one showing modal pupups in `begin_popup_modal`